MantleTests/MTLJSONAdapterSpec.m

```diff
@@ -106,6 +106,34 @@ + (NSValueTransformer *)NSDateJSONTransformer {
 	expect(serializationError).to(beNil());
 });
 
+it(@"it should initialize properties with keys that contain dot literals from JSON", ^{
+	NSDictionary *values = @{
+							 @"literal.name": @"foo",
```

Comment:
Can we ident this more pleasantly? (I understand this is the default output that Xcode produces…)